pkg/manifests/assets/router/deployment.yaml

```diff
@@ -17,6 +17,7 @@ spec:
         - name: router
           # image is set at runtime.
           imagePullPolicy: IfNotPresent
+          args: ["--blueprint-route-pool-size=0"]
```

Comment:
The operator generally uses environment variables rather than command-line switches.  Any reason not to use the `ROUTER_BLUEPRINT_ROUTE_POOL_SIZE` environment variable for consistency?

Reply:
Yes, the reason is that [`ROUTER_BLUEPRINT_ROUTE_POOL_SIZE` has a lower bound of `1`](https://github.com/openshift/cluster-ingress-operator/pull/1113#issuecomment-2289153747).

Reply:
I can go ahead and modify the router's code to change the lower bound to `0`. This was done to minimize the changes however this may be inconsistent with the other configurations indeed.

Reply:
Will be done in https://github.com/openshift/router/pull/635.

Reply:
Done.